python-2/rps.py

```diff
@@ -0,0 +1,121 @@
+        print("Both players have three choices of moves, Those moves are Rock, Paper, or Scissors.") #instructions
+        time.sleep(2.4)#adds a 2.4 second pause
+        print("Mathmatically this game leaves a random winner. Both players will chose a move.") #instructions
+        time.sleep(2.4)#adds a 2.4 second pause
+        print("Those moves will go against eachother and a winner will be determined.") #instructions
+        time.sleep(2.4)#adds a 2.4 second pause
+        print("In this game, Rock beats Scissors, Scissors beats Paper, and Paper beats Rock.") #instructions
+        time.sleep(2.4)#adds a 2.4 second pause
+
+        Ready_name = input ("Now that you know how this game is played, Are you ready to play?\n") #asks if the player is now ready to start
+        
+
+
+    print("Awesome! Then lets get started!") #Says lets start
+    time.sleep(2) #adds a 2 second pause
+
+
+    Winner = False #sets up a true false statement to see if a winner has been had yet
+
+
+    while Winner == False: #while loop saying if their isnt a winner, then run.
+        print("Would you like to play Rock, Paper, or Scissors!\n") #asks player for their move
+        time.sleep(0.5) #adds 0.5 second pause
+        player_move = input ("Your Move:") #prompts for players move
+        
+        PM = 0 #sets player move to 0 or not yet a valid move
+        PMW = ("") #sets their move word to nothing yet
+        if player_move in ["Rock","rock","R","r","ROCK","rox"]: #if players move is in these options, set their word to Rock! and move number to 1 which we use as rock
+            PMW = ("Rock!")
+            PM = 1
+        elif player_move in ["Paper","paper","P","p","PAPER","pap"]: #if players move is in these options, set their word to Paper! and move number to 2 which we use as paper
+            PMW = ("Paper!")
+            PM = 2
+        elif player_move in ["Scissors","scissors","S","s","SCISSORS"]: #if players move is in these options, set their word to Scissors! and move number to 3 which we use as scissors
+            PMW = ("Scissors!")
+            PM = 3
+        elif player_move not in [""]: #if the players move isnt in these lists, then their move is invalid and will wait or continue until a valid move is entered
+            continue
+        
+
+        print(f"You choose to play {player_move}!") #confirms what move was chosen
+        time.sleep(2) #adds a 2 second pause
+        print("Lets see who wins!") #prompts the game start to see who will win
+
+
+        time.sleep(2) #adds a 2 second pause
+        print("Rock!") #prints game chant 
+        time.sleep(1) #adds a 1 second pause
+        print("Paper!")#prints game chant 
+        time.sleep(1) #adds a 1 second pause
+        print("Scissors!")#prints game chant 
+        time.sleep(1) #adds a 1 second pause
+        print("Shoot!\n")#prints game chant 
+        time.sleep(1) #adds a 1 second pause
+
+
+        CM = 0 #sets computer move number to 0 or nothing
```

Comment:
CM/CMW/PW should be more descriptive, I can see that it means computer move/player move but it is better to have a more descriptive name.

Reply:
Also uppercase variables are typically used when they are constants.